### ISSUE TITLE
training loss 

### ISSUE THREAD

**Issue by caoyangcr7** [akarshzingade/image-similarity-deep-ranking#28]
Thanks for your work firstly. 
when I train my own dataset---10 types of bmw cars, totally 512 images  (the url is http://ai.stanford.edu/~jkrause/cars/car_dataset.html), but the loss nearly can't be cut down when I using your original code or I changed the code like https://github.com/akarshzingade/image-similarity-deep-ranking/issues/17.
That's confused...
Anyone meets the same issue with me ?
Thanks for your reply.
